inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -143,6 +143,27 @@ INSTANTIATE_TEST_CASE_P(smoke_ConvolutionBackpropData2D_AutoPadding_OutputPaddin
                                 ::testing::Values(CommonTestUtils::DEVICE_CPU)),
                         ConvolutionBackpropLayerTest::getTestCaseName);
 
+INSTANTIATE_TEST_CASE_P(smoke_ConvolutionBackpropData2D_RuntimeError, ConvolutionBackpropLayerTest,
+                        ::testing::Combine(
+                                ::testing::Combine(
+                                        ::testing::ValuesIn(std::vector<std::vector<size_t >>({ {31, 1} })),
+                                        ::testing::ValuesIn(std::vector<std::vector<size_t >>({ {2, 1} })),
+                                        ::testing::Values(std::vector<ptrdiff_t>({14, 0})),
+                                        ::testing::Values(std::vector<ptrdiff_t>({15, 0})),
+                                        ::testing::ValuesIn(std::vector<std::vector<size_t >>({ {1, 1} })),
+                                        ::testing::ValuesIn(std::vector<size_t>({ 512 })),
+                                        ::testing::Values(ngraph::op::PadType::EXPLICIT),
+                                        ::testing::ValuesIn(std::vector<std::vector<ptrdiff_t >>({ {0, 0} }))),
+                                ::testing::ValuesIn(netPrecisions),
+                                ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                ::testing::Values(InferenceEngine::Layout::ANY),
+                                ::testing::Values(InferenceEngine::Layout::ANY),
+                                ::testing::Values(std::vector<size_t >({ 1, 256, 4, 1 })),
```

Comment:
Same comment regarding number of channels

Reply:
Done a7941e5e69ca3d8cb33e992b4d296ac9da6f030c